pinot-common/src/main/java/org/apache/pinot/common/datablock/BaseDataBlock.java

```diff
@@ -442,12 +425,6 @@ public byte[] toBytes()
     DataOutputStream dataOutputStream = new DataOutputStream(byteArrayOutputStream);
     writeLeadingSections(dataOutputStream);
 
-    // Add table serialization time metadata if thread timer is enabled.
```

Comment:
Hey @walterddr is removing this part intended?

Reply:
yes. 
1. we no longer encode metadata block together with data
2. data table related code are moved to DataTableImplV4